[2024-05-19T20:44:29.132+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-19T20:44:29.141+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: disney_s3.process_and_save_to_s3 manual__2024-05-19T20:44:25.621849+00:00 [queued]>
[2024-05-19T20:44:29.144+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: disney_s3.process_and_save_to_s3 manual__2024-05-19T20:44:25.621849+00:00 [queued]>
[2024-05-19T20:44:29.145+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-19T20:44:29.152+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): process_and_save_to_s3> on 2024-05-19 20:44:25.621849+00:00
[2024-05-19T20:44:29.156+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3217) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-19T20:44:29.157+0000] {standard_task_runner.py:63} INFO - Started process 3220 to run task
[2024-05-19T20:44:29.157+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'disney_s3', 'process_and_save_to_s3', 'manual__2024-05-19T20:44:25.621849+00:00', '--job-id', '432', '--raw', '--subdir', 'DAGS_FOLDER/disney_s3.py', '--cfg-path', '/tmp/tmp42rna1km']
[2024-05-19T20:44:29.159+0000] {standard_task_runner.py:91} INFO - Job 432: Subtask process_and_save_to_s3
[2024-05-19T20:44:29.198+0000] {task_command.py:426} INFO - Running <TaskInstance: disney_s3.process_and_save_to_s3 manual__2024-05-19T20:44:25.621849+00:00 [running]> on host 620090466bc8
[2024-05-19T20:44:29.239+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='disney_s3' AIRFLOW_CTX_TASK_ID='process_and_save_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-05-19T20:44:25.621849+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-19T20:44:25.621849+00:00'
[2024-05-19T20:44:29.240+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-19T20:44:29.254+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-05-19T20:44:29.292+0000] {base.py:84} INFO - Using connection ID 'aws_s3' for task execution.
[2024-05-19T20:44:29.293+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_s3', conn_type='aws') credentials retrieved from login and password.
[2024-05-19T20:44:32.945+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-19T20:44:32.946+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-19T20:44:32.951+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=disney_s3, task_id=process_and_save_to_s3, run_id=manual__2024-05-19T20:44:25.621849+00:00, execution_date=20240519T204425, start_date=20240519T204429, end_date=20240519T204432
[2024-05-19T20:44:32.965+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-19T20:44:32.975+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-19T20:44:32.976+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
